components/input/docs/partials/api.md

```diff
@@ -157,6 +157,25 @@ The `max` attribute should be used in combination with the `setCustomValidityRan
 
 </auro-accordion>
 
+#### Max Length <a name="maxLength"></a>
+
+Use the `maxlength` attribute to control the length of the input entered.
+
+The `minLength` attribute should be used in combination with the `setCustomValidityTooLong` attribute to define help text used when the length of the input is too long.
```

Comment:
I wouldn't call out the validity part here as "should be used in combination" because we already show the browser default message that is localized for this error. The `setCustomValidityTooLong` attribute is purely optional.

Also, I think you meant `maxLength` here instead of `minLength`.

Reply:
FWIW, if we are going to add commentary about which attribute to use to customize the validationState error for any requirement attribute we should do it for all of them.

Reply:
I added this to follow suit with what we are currently doing on the docsite. You can see us suggesting them to use the `setCustomValidity` states under the "Pattern", "Max" and "Min" examples on the API page.